server/src/test/scala/services/OptimiserPlusSpec.scala

```diff
@@ -0,0 +1,175 @@
+package services
+
+import org.specs2.mutable.Specification
+
+import scala.util.Try
+
+
+class OptimiserPlusSpec extends Specification {
+  "Given 1 minutes incoming workload per minute, and desks fixed at 1 per minute" >> {
+    "I should see all the workload completed each minute, leaving zero wait times" >> {
+      val result: Try[OptimizerCrunchResult] = OptimiserPlus.crunch(Seq.fill(30)(1), Seq.fill(30)(1), Seq.fill(30)(1), OptimiserConfig(20, EGateWorkloadProcessors(Iterable(1))))
```

Comment:
it's quite hard to see what's going on in these tests, perhaps we could extract the crunch parameters to appropriately named vals?

Reply:
Fair point. I'll extract them :+1: